system-addon/content-src/components/ASRouterAdmin/ASRouterAdmin.scss

```diff
@@ -7,7 +7,7 @@
   font-size: 14px;
   // Reset .outer-wrapper styles
   display: inherit;
-  padding: 0 0 42px;
+  padding: 0 0 92px;
```

Comment:
this is just a little adjustment so you can see the bottome of the admin when there are a lot of messages